cns/azure-cns-windows.yaml

```diff
@@ -38,14 +38,12 @@ spec:
           command: ["powershell.exe"]
           args: 
             [ 
-              '.\setkubeconfigpath.ps1', ";",
               'powershell.exe', '.\azure-cns.exe',
```

Comment:
drop this one too since `powershell.exe` is the command?

Reply:
nvm, suggest doing the same that I mentioned in the comment on the integration test